.pre-commit-config.yaml

```diff
@@ -28,7 +28,7 @@ repos:
   - id: check-merge-conflict
   - id: check-yaml
 - repo: https://github.com/pycqa/flake8
-  rev: "5.0.4"
```

Comment:
same observation as above? (quote in all the places or quotes in no places?)

Reply:
removed quotes